doc/pages/Manual.md

```diff
@@ -178,6 +178,18 @@ Switches are laid out thusly:
 - `<switch-prefix>/.opam-switch/backup`: snapshots of previous states of the
   switch, and other backup files.
 
+#### Plugins
+
+When opam doesn't know a subcommand name (e.g. `opam plugin-name`), it looks at
+the binaries installed in `${OPAMROOT}/plugins/bin` (by default: `~/.opam/plugins/bin`).
+This directory contains symlinks to binaries installed in whichever opam switch
+the plugin it belongs to was installed in.
+These symlinks are added for any packages that have the ["plugin" flag](#opamflag-plugin)
+and have a binary matching their package name. If no binaries with the same name
```

Comment:
I think the binary only has to match the opam package name by convention; it's not a hard requirement.

Reply:
it is a "hard" requirement. A warning will be displayed otherwise and the link will not appear.